src/compiler/CompiledMediaType.ts

```diff
@@ -14,6 +14,7 @@ export default class CompiledMediaType {
   public validate(value: any) {
     try {
       this.compiledSchema.validate(value);
+      return value;
```

Comment:
Is Ajv gonna magically mutate the value and coerce it to the right type?

Reply:
Yes that's correct. It does it when the coerceTypes option is true. In this specific case the option is not true, so this just returns the original data without modification. But I changed the function anyway to be consistent with CompiledParameterPath and CompiledParameterQuery where the coerceTypes option is provided.